packages/esm-amd-loader/README.md

```diff
@@ -2,7 +2,7 @@
 
 # @polymer/esm-amd-loader
 
-A JavaScript library which loads AMD-style modules in the browser in 0.9 KB.
+A JavaScript library which loads AMD-style modules in the browser in 1.2 KB.
```

Comment:
Size is also mentioned below. Also you might want to update some of the documentation with the renamed things, e.g. factory is gone.